include/psen_scan_v2/scanner_v2.h

```diff
@@ -76,7 +76,7 @@ class ScannerV2 : public IScanner
   void stopStartWatchdog();
 
 private:
-  std::promise<void> scanner_has_started_;
+  std::vector<std::promise<void>> scanner_has_started_;
```

Comment:
Only `ScannerV2::start()` is considered, however, the problem is the same for the `ScannerV2::stop()`.